app/models/merritt/feed.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Merritt
+  class Feed < ActiveRecord::Base
+    def self.table_name_prefix
+      "merritt_"
+    end
+
+    HOME = "https://merritt.cdlib.org"
+
+    def self.etd_feed_url(page = 1)
+      HOME + "/object/recent.atom?collection=ark:/13030/m5pv6m0x&page=#{page}"
+    end
+
+    def self.parse(page = 1)
+      url = etd_feed_url(page)
+      uri = URI.parse(url)
+      request = Net::HTTP::Get.new(uri.request_uri)
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+      response = http.request request
+
+      if response.code.to_i != 200
+        raise StandardError,
+              "Error in Merritt::Feed.parse:"\
+              "#{url} returned #{response.code} #{response.message}"
+      end
+
+      xml = response.body
+      Feedjira.parse(xml)
```

Comment:
Why can't we use Nokogiri for this?

Reply:
I liked Feedjira because it has built-in functionality for parsing Atom feeds. It is super fast as well.